### PR TITLE
Add automated tests with jest

### PR DESCRIPTION
**Pull request by noahpryor**
Haven't covered all the test cases yet but its a start
- Adds `jest` for automated testing
- Sets up circleci

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "A JavaScript plugin to warn users about links to private pages.",
   "main": "private-eye.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "jest"
```
Comment (afeld) [inline package.json]: Mind adding a note to the README or CONTRIBUTING files about how to run the tests?

**Review by afeld — CHANGES_REQUESTED** [review]

**Comment by afeld** [comment]
Closes #4.

File: package.json (hunk 1 above)
Comment (noahpryor) [reply package.json]: Done - added to README.md, let me know if there's a better way to phrase it/place, it feels a little tacked on

**Comment by noahpryor** [comment]
Totally open to any feedback on test structure/it statements etc, I'm definitely new to the JS testing world

**Comment by afeld** [comment]
You rock, thanks!

**Pull request merged**